<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DSS::DropFilesDlg</class>
 <widget class="QDialog" name="DSS::DropFilesDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add files ...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="dropFiles">
     <property name="title">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="lightFrames">
        <property name="text">
         <string notr="true">Light Frames</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="darkFrames">
        <property name="text">
         <string notr="true">Dark Frames</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="flatFrames">
        <property name="text">
         <string notr="true">Flat Frames</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="darkFlatFrames">
        <property name="text">
         <string notr="true">Dark Flat Frames</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="biasFrames">
        <property name="text">
         <string notr="true">Offset/Bias Frames</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
